### PR TITLE
Add Large File Support

### PR DESCRIPTION
**Pull request by nick-benoit14**
* Disables timeout for *.imscc upload 
* Adds support for 64 bit zip

### CODE REVIEW

File: lib/senkyoshi.rb [hunk 1]
```diff
@@ -11,6 +11,8 @@
 require "nokogiri"
 require "zip"
 
+Zip.write_zip64_support = true
```
Comment (bfcoder) [inline lib/senkyoshi.rb:14]: Just my opinion, but I feel this is not in the scope of this PR and should be its own PR.

Comment (nick-benoit14) [reply lib/senkyoshi.rb:14]: What if we changed the name of the PR to 'Add Large File Support'? Then I think it better reflects the reason we are changing these things.

Comment (bfcoder) [reply lib/senkyoshi.rb:14]: Then it would be in scope. :] 👍

**Review by srphillips — APPROVED** [review]

**Comment by bfcoder** [comment]
I would suggest we make the timeout configurable with the yml file. Have it default to 30 mins perhaps?

**Comment by srphillips** [comment]
@bfcoder makes an excellent suggestion to put it into the config

File: lib/senkyoshi/configuration.rb [hunk 2]
```diff
@@ -16,6 +17,7 @@ def initialize
       @scorm_url = Configuration._config[:scorm_url]
       @scorm_launch_url = Configuration._config[:scorm_launch_url]
       @scorm_shared_auth = Configuration._config[:scorm_shared_auth]
+      @request_timeout = Configuration._config[:request_timeout]
```
Comment (bfcoder) [inline lib/senkyoshi/configuration.rb]:
```ruby
DEFAULT_TIMEOUT = 1_800 # 30 minutes
```
```ruby
@request_timeout = Configuration._config[:request_timeout] || DEFAULT_TIMEOUT
```

File: lib/senkyoshi/configuration.rb [hunk 3]
```diff
@@ -16,6 +17,9 @@ def initialize
       @scorm_url = Configuration._config[:scorm_url]
       @scorm_launch_url = Configuration._config[:scorm_launch_url]
       @scorm_shared_auth = Configuration._config[:scorm_shared_auth]
+
+      # Default timeout is 30 minutes
+      @request_timeout = Configuration._config[:request_timeout] || 1800
```
Comment (bfcoder) [inline lib/senkyoshi/configuration.rb]:
```ruby
TIMEOUT = 1_800 # 30 minutes
```
```ruby
@request_timeout = Configuration._config[:request_timeout] || TIMEOUT
```

**Review by bfcoder — APPROVED** [review]
Looks great!

**Comment by bfcoder** [comment]
wait, still needs the readme to be updated

**Review by bfcoder — APPROVED** [review]
excellent
